### ISSUE TITLE
Enforce unique constraint on show/email pair on RSVPs

### ISSUE THREAD

**Issue by paulschreiber** [paulschreiber/houseconcerts#5]
Enforce a unique constraint on show/email pair on RSVPs:
* Clean up existing data

* Add a migration
```
def change
   add_index :rsvps, [:show_id, :email], unique: true
end
```

**Comment by paulschreiber** [comment]
```
CREATE TABLE rsvps_new LIKE rsvps;
ALTER TABLE rsvps_new ADD UNIQUE INDEX (email, show_id);
INSERT IGNORE INTO rsvps_new SELECT * FROM rsvps;
DROP TABLE rsvps;
RENAME TABLE rsvps_new to rsvps;
```

**Comment by paulschreiber** [comment]
Fixed with 17b116df754af4a5b473ef4c3543546adc36e2a4.
